Gemfile

```diff
@@ -3,12 +3,9 @@ gemspec
 
 gem 'rest-client', :github => 'chef/rest-client'
 
-gem 'oc-chef-pedant', :github => 'chef/chef-server'
-
-# gem 'oc-chef-pedant', :path => "../chef-server"
+gem 'oc-chef-pedant', :github => 'chef/chef-server', :branch => "jk/authorization-tags"
 
 # bundler resolve failure on "rspec_junit_formatter"
 # gem 'chef-pedant', :github => 'opscode/chef-pedant', :ref => "server-cli-option"
 
-gem 'chef', :github => 'chef/chef'
-# gem 'chef', :path => "../chef"
+gem 'chef', :github => 'chef/chef', :branch => 'jk/policies-acls'
```

Comment:
Actually, never mind. Not that important, we're good at remembering these things now.